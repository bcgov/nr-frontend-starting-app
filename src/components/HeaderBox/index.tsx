import styled from 'styled-components';

const HeaderBox = styled.header`
  background-color: #036;
  border-bottom: 2px solid #fcba19;
  align-items: center;
  padding: 5px 65px 0 65px;
  color: #fff;
  display: flex;
  height: 65px;
  top: 0px;
  position: fixed;
  width: 100%;
  z-index: 1000;
`;

export default HeaderBox;
