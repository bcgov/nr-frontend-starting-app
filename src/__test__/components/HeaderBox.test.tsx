/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import HeaderBox from '../../components/HeaderBox';

// This component serves only as a 'parent' to style correctly the header
// so the snapshot test is sufficient

describe('the HeaderBox component', () => {
  it('should match the snapshot', () => {
    const tree = renderer
      .create(<HeaderBox />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
