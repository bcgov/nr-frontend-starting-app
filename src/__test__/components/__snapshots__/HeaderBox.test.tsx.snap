// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the HeaderBox component should match the snapshot 1`] = `
<header
  className="sc-bczRLJ ksKPAx"
/>
`;
